packages/libs/react-ui/src/components/Form/FormFieldWrapper/FormFieldWrapper.tsx

```diff
@@ -0,0 +1,57 @@
+import type { IInputProps } from '@components/Form';
+import type { FC, FunctionComponentElement } from 'react';
+import React from 'react';
+import type { vars } from 'src/styles';
+import type { FormFieldStatus } from '../Form.css';
+
+import { Stack } from '@components/Layout';
+import type { IFormFieldHeaderProps } from './FormFieldHeader/FormFieldHeader';
+import { FormFieldHeader } from './FormFieldHeader/FormFieldHeader';
+import { FormFieldHelper } from './FormFieldHelper/FormFieldHelper';
+import { FormFieldWrapperContext } from './FormFieldWrapper.context';
+import { statusVariant } from './FormFieldWrapper.css';
+
+export interface IFormFieldWrapperProps
+  extends Omit<IFormFieldHeaderProps, 'label'> {
+  children:
+    | FunctionComponentElement<IInputProps>
```

Comment:
@eileenmguo here we use the IInputProps but it should be | ITextareaProps | ... right?

Reply:
This is ensuring it accepts a lone input or input array, but yes, we should add the additional form elements here